docs/contributing/testing.rst

```diff
@@ -149,7 +149,7 @@ The file ``tests/bin/test-setup.sh`` is the best reference to environment
 variables that can affect the tests' behavior. Here are some important ones:
 
 - ``HOST_IPADDR`` - address on which Docker containers can communicate for the
-  functional tests, probably the host's IP or the one assigned to boot2docker_.
+  functional tests, probably the host's IP or the one assigned to `Docker Toolbox`_.
```

Comment:
Use `docker-machine` instead of Docker toolbox, docker toolbox is just the installer.